deployments/kustomize/base2/database-migration-job.yaml

```
@@ -0,0 +1,37 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: database-migration
+  annotations:
+    app.kubernetes.io/name: database-migration
+    argocd.argoproj.io/hook: PostSync
+    argocd.argoproj.io/hook-delete-policy: HookSucceeded
+spec:
+  template:
+    spec:
+      restartPolicy: OnFailure
+      containers:
+        - name: naad-database-migration
+          image: bcgovgdx/naad-app
```

```suggestion
          image: image-registry.openshift-image-registry.svc:5000/bdf349-tools/naad-app
```
Should match the image used in the stateful set I think. I assume it was already pointing to the wrong image before any of these changes.

---

actually the stateful set needs to be updated, we shouldn't have the final image here, we should let kustomize update this, and if `bcgovgdx/naad-app` then it is easier